axon-server-connector/src/test/java/org/axonframework/axonserver/connector/heartbeat/FakeClock.java

```diff
@@ -0,0 +1,43 @@
+package org.axonframework.axonserver.connector.heartbeat;
+
+import java.time.Clock;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.util.function.Supplier;
+
+/**
+ * @author Sara Pellegrini
+ * @since 4.2
```

Comment:
Nit: I believe we do not use the `since` tag for test classes. Not sure what convention would be best in this respect.

Reply:
I've removed the tag